bot/cogs/snekbox.py

```diff
@@ -101,86 +85,140 @@ def rmq(self) -> RMQ:
             log.trace(f"Extracted {info} for evaluation:\n{code}")
         else:
             code = textwrap.dedent(RAW_CODE_REGEX.fullmatch(code).group("code"))
-            log.trace(f"Eval message contains not or badly formatted code, stripping whitespace only:\n{code}")
+            log.trace(
+                f"Eval message contains unformatted or badly formatted code, "
+                f"stripping whitespace only:\n{code}"
+            )
 
-        code = textwrap.indent(code, "    ")
-        code = CODE_TEMPLATE.replace("{CODE}", code)
+        return code
+
+    @staticmethod
+    def get_results_message(results: dict) -> Tuple[str, str]:
+        """Return a user-friendly message and error corresponding to the process's return code."""
+        stdout, returncode = results["stdout"], results["returncode"]
+        msg = f"Your eval job has completed with return code {returncode}"
```

Comment:
Would it be clearer if it was called an "exit code" instead?

Reply:
No, return code is fine.